pandas/tests/frame/indexing/test_where.py

```diff
@@ -745,3 +745,11 @@ def test_where_bool_comparison():
         }
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_where():
+
+    s = Series(range(5))
+
+    with tm.assert_produces_warning(FutureWarning):
+        s.where(s > 1, 10, False)
```

Comment:
A few comments here:

1. this is in pandas/tests/frame/index/test_where.py, but here we have a series - could you make this test use a DataFrame instead?
2. could you check the warning message that the test produces? See the PR I linked to for an example
3. Could you also check the result? So here it would be
    ```
    with tm.assert_produces_warning(FutureWarning):
        result = s.where(s > 1, 10, False)
    expected = Series([10, 2, 3])
    tm.assert_series_equal(result, expected)
   ```
4. Could you give the test a more descriptive title?
5. In the line under `test_where`, could you put a comment with the issue number of the issue this is addressing?